kwmserver/wrapper.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+if [ "$INSECURE" = "yes" ]; then
+	exec kwmserverd serve \
+		--insecure \
+		--iss="https://$oidc_issuer_identifier"
+else
+	exec kwmserverd serve \
+		--iss="https://$oidc_issuer_identifier"
+fi
```

Comment:
This if  / else is a little complicated and duplicates code. See the binscripts in scripts directory of the source tree for konnect, kwmserver or kweb on how to extend args based on environment variables without duplicating lines.

Reply:
updated